sdk/ports/_template/az_platform_impl.h

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_PLATFORM_IMPL_H
+#define _az_PLATFORM_IMPL_H
+
+#include <azure/core/az_result.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <azure/core/_az_cfg.h>
+
+/**
+ * @brief Gets the platform clock in milliseconds.
+ * @return Platform clock in milliseconds.
+ */
+AZ_NODISCARD AZ_INLINE int64_t az_platform_clock_msec()
+{
+// The method must be implemented
+// The Azure SDK for Embedded C provides default implementations for Win32, Linux and Mac.
+//  https://github.com/Azure/azure-sdk-for-c#cmake-options
+#error \
+    "No method implementation provided, see documentation for additional details.  https://github.com/Azure/azure-sdk-for-c"
```

Comment:
should this be
```suggestion
    "No method implementation provided, see documentation for additional details.  https://github.com/Azure/azure-sdk-for-c#cmake-options"
```

Reply:
It doesn't render well in the output with or without it

Reply:
Should it actually be a link to:
https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/platform#platform

Reply:
With maybe some additions to that section describing what needs to be done. I worry about the link that is currently there because it gives no specificity to the problem they are getting.